mvcc/kvstore.go

```diff
@@ -256,8 +255,11 @@ func (s *store) Compact(rev int64) (<-chan struct{}, error) {
 	// ensure that desired compaction is persisted
 	s.b.ForceCommit()
 
-	s.mu.Unlock()
 	s.revMu.Unlock()
+	return start, nil, nil
+}
+
+func (s *store) compact(rev int64, start time.Time) (<-chan struct{}, error) {
```

Comment:
remove the start time.Time in the arg: compact(rev int64).

get the start time in the func instead.

```go
func (s *store) compact(rev int64) (<-chan struct{}, error) {
    start := time.Now()
    ...
}
```